packages/components/dialog/src/dialog.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <teleport to="body" :disabled="!appendToBody">
+  <teleport v-if="isClient" to="body" :disabled="!appendToBody">
```

Comment:
According to the nuxt documentation, it seems to support teleport to `body`.

Reply:
It may be necessary to verify that native(Vue SSR) also supports

Reply:
If using the native SSR to render multiple content `teleport` to the `body`, the content will be displayed in a DOM.

Reply:
OK, I see